src/chocolatey.resources/redirects/RefreshEnv.cmd

```diff
@@ -1,27 +1,43 @@
 @echo off
-::
-:: RefreshEnv.cmd
-::
-:: Batch file to read environment variables from registry and
-:: set session variables to these values.
-::
-:: With this batch file, there should be no need to reload command
-:: environment every time you want environment changes to propagate
+setlocal enableextensions enabledelayedexpansion
+REM RefreshEnv.cmd
 
-::echo "RefreshEnv.cmd only works from cmd.exe, please install the Chocolatey Profile to take advantage of refreshenv from PowerShell"

```

Comment:
It appears this commented out bit was removed. Was wanting to keep this here for now.

Reply:
I see you've rewritten it below as an actual comment. I guess that is fine for now.

Reply:
replaced ::s with REM to avoid "the system cannot find the drive specified" errors

Reply:
That makes sense